ProjectPlant-HW/Simulator/Simulator.ino

```diff
@@ -0,0 +1,74 @@
+#include <SoftwareSerial.h>
+#define VERSION "SIM0.1"
+#define BLUETOOTH_RX 11
+#define BLUETOOTH_TX 10
+#define BLUETOOTH_BAUD 9600
+
+char buf[30];
+int temp = 25;
+int humidity = 65;
+int light = 50;
+int moisture = 50;
+
+SoftwareSerial bluetooth(BLUETOOTH_RX, BLUETOOTH_TX);
+
+void setup()
+{
+  bluetooth.begin(BLUETOOTH_BAUD);
+}
+
+void adjustValues()
+{
+  int adjustTemp = random(0,100);
+  int adjustHumidity = random(0,100);
+  int adjustLight = random(0, 100);
+  int adjustMoisture = random(0, 100);
+
+  adjustValue(temp, adjustTemp);
+  adjustValue(humidity, adjustHumidity);
+  adjustValue(light, adjustLight);
+  adjustValue(moisture, adjustMoisture);
+}
+
+void adjustValue(int &adjustVariable, int &adjustPressure)
+{
+  if (adjustPressure < 33)
+  {
+    adjustVariable = adjustVariable - 2;
+  }
+  else if (adjustPressure > 66)
+  {
+    adjustVariable = adjustVariable + 2;
+  }
```

Comment:
I would probably go with something like 
```
if (adjustPressure < 33) adjustVariable -= 2;
else adjustVariable += 2;
``` 
but since its a tester I'm ok with the current state